playbooks/common/openshift-master/config.yml

```diff
@@ -351,7 +351,7 @@
   - role: openshift_cluster_metrics
     when: openshift.common.use_cluster_metrics | bool
   - role: openshift_manageiq
-    when: openshift.common.use_manageiq | bool
+    when: openshift.common.version_greater_than_3_1_or_1_1 | bool and openshift.common.use_manageiq | bool
```

Comment:
I would probably skip the version_greater_than_3_1_or_1_1 test here and instead have the openshift_manageiq role fail if version_greater_than_3_1_or_1_1 is false.

It might require adding the openshift_facts role as a dependency in the openshift_manageiq role if it isn't already (meta/main.yml in the role dir).